packages/router/router.coffee

```diff
@@ -4,3 +4,15 @@ FlowRouter.route '/',
     FlowLayout.render 'layout',
       main: 'splashPage'
       params: {}
+
+FlowRouter.route '/signIn',
+  name: 'signIn'
+  action: () ->
+    FlowLayout.render 'layout',
+      main: 'signIn'
```

Comment:
I'd like to see what it would look like to render the sign in template without changing the route.